src/System.Net.Security/tests/FunctionalTests/NegotiateStreamTestForUnix.cs

```diff
@@ -46,7 +46,7 @@ public KDCSetup()
                         Dispose();
                     }
 
-                    Assert.True(false, "KDC setup failure");
+                    throw new IllegalStateException("KDC setup failure");
```

Comment:
.... well, the overall compile succeeds, but only because I wasn't compiling _this file_; I didn't notice it was flagged for unix, and I'm on windows (and wasn't expecting setup to usually throw errors anyways).  So my usual build/run check succeeded.  Changing to compiling for Linux throws compile errors as expected.  Switching to `InvalidOperationException`.

(`IllegalStateException` is from Java, guess it's not only Japanese and Spanish I mix...)